exporter/datadogexporter/stats.go

```diff
@@ -27,13 +27,16 @@ const (
 )
 
 // ComputeAPMStats calculates the stats that should be submitted to APM about a given trace
-func computeAPMStats(tracePayload *pb.TracePayload, calculator *sublayerCalculator, pushTime int64) *stats.Payload {
+func computeAPMStats(tracePayload *pb.TracePayload, pushTime int64) *stats.Payload {
 	statsRawBuckets := make(map[int64]*stats.RawBucket)
 
+	// removing sublayer calc as part of work to port
+	// https://github.com/DataDog/datadog-agent/pull/7450/files
+	emptySublayer := make([]stats.SublayerValue, 0)
```

Comment:
To have a `nil` value like you mention on the PR description we should do
```suggestion
	var emptySublayer []stats.SublayerValue
```